first_module/src/second.java

```diff
@@ -0,0 +1,9 @@
+public class second {
```

Comment:
Due to Java codestyle you should name the class starting with capital letter (`Second`)
Also, I would ask you to think about a name that will describe what this class does